casper/src/main/resources/MakeMint.rho

```diff
@@ -61,12 +68,16 @@ in {
             } |
             contract thisPurse(@"deposit", @amount, @src, success) = {
               new result in {
-                @src!(bundle0{bundle0{src}|*decr}, amount, *result) | //call src decr function.
-                for(@decrSuccess, @auth <- result) {
-                  if (decrSuccess and auth == *decr) {
-                    balance!("add", amount, *success) // add transferred amount to this purse
-                  } else {
-                    success!(false) 
+                for (logCh <<- logStore) {
+                  @src!(bundle0{bundle0{src}|*decr}, amount, *result) | //call src decr function.
+                  for(@decrSuccess, @auth <- result) {
+                    if (decrSuccess and auth == *decr) {
+                      if (Nil != *logCh) { logCh!(["deposit", amount, true]) } |
```

Comment:
`logCh` scope can be inside the scope of `for`.

Reply:
will change.

Reply:
done